plugins/com.google.cloud.tools.eclipse.appengine.flex.test/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -4,5 +4,4 @@ org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.7
 org.eclipse.jdt.core.compiler.compliance=1.7
 org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
 org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
-org.eclipse.jdt.core.compiler.problem.forbiddenReference=warning
```

Comment:
This should be an error not a warning i.e. org.eclipse.jdt.core.compiler.problem.forbiddenReference=error